lib/bolt/node/winrm.rb

```diff
@@ -337,7 +518,23 @@ def _run_task(task, input_method, arguments)
           # NOTE: cannot redirect STDIN to a .ps1 script inside of PowerShell
           # must create new powershell.exe process like other interpreters
           # fortunately, using PS with stdin input_method should never happen
-          output = execute("try { &""#{remote_path}"" } catch { exit 1 }")
+          output = execute(<<-PS)
+& { # create a new scope
+  $_task_path = [Text.Encoding]::UTF8.GetString(
+    [Convert]::FromBase64String('#{Base64.encode64(remote_path)}')
```

Comment:
When would the `remote_path` need to be base64 encoded / decoded? If this is a concern, we should probably have a test demonstrating the problem...

Also a better way to avoid escaping rules with the literal (assuming it's even necessary) would be to just use a here-string like:

```powershell
$taskPath = @'
#{remote_path}
@'
```

Reply:
What about character encoding: UTF8 vs. UTF16?
By using here-string, you lock yourself to the encoding assumed by powershell when parsing the source.
But it my not be a concern here.

Reply:
I more less reverted the original handling of the `remote_path`, except I fixed the quoting, so now we have:
```ruby
"try { & '#{remote_path}' } catch { exit 1 }"
```
instead of the original:
```ruby
"try { &""#{remote_path}"" } catch { exit 1 }"
```